### ISSUE TITLE
[Scope Chains]: Error calling the module

### ISSUE THREAD

**Issue by mesteves** [nodeschool/discussions#1189]
Hi,

After the instalation of the module - npm install -g scope-chains-closures - and executing the command  - scope-chains-closures - i have the following message:

```
C:\Users\...\Desktop\Node App\scope-chain-closures>scope-chains-closures
The system cannot find the path specified.
```

When i list the globally installed packages the scc is there:

```
C:\Users\...\Desktop\Node App\scope-chain-closures>npm list -g --depth=0
C:\Users\...\AppData\Roaming\npm
├── how-to-npm@2.0.0
├── learnyounode@2.5.2
├── promise-it-wont-hurt@0.1.0
└── scope-chains-closures@1.0.2
```

As you can see, i've also installed other modules, and when i execute 'how-to-npm' or any other, i don't have any problem.

Did i miss anything?

Thanks in advance,
Manuel

**Comment by mesteves** [comment]
Hi,

I was able to solved it by changing the cmd file:

from 
```
@IF EXIST "%~dp0\/bin/sh.exe" (
  "%~dp0\/bin/sh.exe"  "%~dp0\node_modules\scope-chains-closures\runner.js" %*
) ELSE (
  /bin/sh  "%~dp0\node_modules\scope-chains-closures\runner.js" %*
)
```

to
```
@IF EXIST "%~dp0\node.exe" (
  "%~dp0\node.exe"  "%~dp0\node_modules\scope-chains-closures\runner.js" %*
) ELSE (
  node  "%~dp0\node_modules\scope-chains-closures\runner.js" %*
)
```

**Comment by martinheidegger** [comment]
:+1: Awesome that it works for you! But your fix is not "a good one" since it will break for a few other packages as well. This seems to be a windows related problem to the way the [shebang](https://github.com/jesstelford/scope-chains-closures/blob/9ad39247b2d87a99979688c135ade8cc413d9521/runner.js#L2) is implemented in scope-chains. Would you mind opening an issue in https://github.com/jesstelford/scope-chains-closures/issues telling them that its related to your windows installation?

**Comment by mesteves** [comment]
Thank you Martin.

I've referenced the problem in a issue that is already open.

:+1:

**Comment by jesstelford** [comment]
@mesteves My apologies! A fix has come in, and it is now published in **v1.0.3**.
To update: `npm install -g scope-chains-closures`.

Thanks for the help @martinheidegger :)
